Orthtree/include/CGAL/Orthtree.h

```diff
@@ -782,7 +782,28 @@ class Orthtree {
    */
   template <typename Query, typename OutputIterator>
   OutputIterator intersected_nodes(const Query& query, OutputIterator output) const {
-    return intersected_nodes_recursive(query, root(), output);
+    return intersected_nodes_recursive(query, root(), output, [](const Query& query, const typename Traits::Bbox_d &box) -> bool {return CGAL::do_intersect(query, box);});
```

Comment:
I believe we should provide an API for an intersection with a ball, as it is a common operation, say `intersected_nodes(query, center, radius, output)`